frontend/src/services/documentService.ts

```diff
@@ -1,5 +1,8 @@
+import { v4 } from 'uuid';
```

Comment:
Usually a safer consideration to have this be aliased to something like uuidv4 or similar so that we know at a glance what the intent is.